spec/support/upload_params.cr

```diff
@@ -0,0 +1,16 @@
+class Avram::UploadParams < Avram::Params
```

Comment:
I tried changing `Avram::Params` first, but kept running into many issues because it's used in many places. That's why I decided to subclass it, just to test uploads specifically.